omero/training-server/playbook.yml

```diff
@@ -255,6 +257,8 @@
     omero_web_release: "{{ omero_web_release_override | default('5.4.6') }}"
     omero_figure_tag: "{{ omero_figure_tag_override | default('3.2.1') }}"
     omero_fpbioimage_tag: "{{ omero_fpbioimage_tag_override | default('0.2.0') }}"
+    omero_mapr: "{{ omero_mapr_override | default('0.2.3') }}"
```

Comment:
to be consistent it should be ``omero_mapr_tag`` and ``omero_parade_tag``

Reply:
maybe even use `_release` everywhere to match the semantics defined in the `omero-{server,web}` roles